logger.go

```diff
@@ -349,3 +354,9 @@ func (logger *Logger) ReplaceHooks(hooks LevelHooks) LevelHooks {
 	logger.mu.Unlock()
 	return oldHooks
 }
+
```

Comment:
Please add a godoc comment for this exported function. Something along the lines of:

```go
// SetErrorCallback sets the ErrorCallback function of the logger
```